mars/dataframe/indexing/getitem.py

```diff
@@ -0,0 +1,195 @@
+                return i
+        raise TypeError("label %s doesn't exist" % label)
+
+    @classmethod
+    def tile(cls, op):
+        in_series = op.inputs[0]
+        out_series = op.outputs[0]
+        if not isinstance(in_series.index_value._index_value, IndexValue.RangeIndex):
+            raise NotImplementedError
+        chunk_indexes = [c.index_value.to_pandas() for c in in_series.chunks]
+        if not isinstance(op.labels, (tuple, list)):
+            selected_chunk = in_series.chunks[cls._calc_chunk_index(op.labels, chunk_indexes)]
+            index_op = op.copy().reset_key()
+            out_chunk = index_op.new_chunk([selected_chunk], shape=(), dtype=selected_chunk.dtype,
+                                         index_value=parse_index(pd.RangeIndex(0)))
+            new_op = op.copy()
+            return new_op.new_seriess(op.inputs, shape=out_series.shape, dtype=out_series.dtype,
+                                      index_value=out_series.index_value, nsplits=(), chunks=[out_chunk])
+        else:
+            selected_index = [cls._calc_chunk_index(label, chunk_indexes) for label in op.labels]
+            splits = []
+            start = 0
+            for i in range(len(selected_index)):
+                if i == len(selected_index) - 1 or selected_index[i] != selected_index[i + 1]:
+                    splits.append((op.labels[slice(start, i + 1)], selected_index[start]))
+                    start = i + 1
+            out_chunks = []
+            nsplits = []
+            for i, (labels, idx) in enumerate(splits):
+                index_op = SeriesIndex(labels=labels)
+                c = in_series.chunks[idx]
+                nsplits.append(len(labels))
+                out_chunks.append(index_op.new_chunk([c], shape=(len(labels),), dtype=c.dtype,
+                                                     index_value=parse_index(pd.RangeIndex(len(labels))),
+                                                     name=c.name, index=(i,)))
+            new_op = op.copy()
+            return new_op.new_seriess(op.inputs, shape=out_series.shape, dtype=out_series.dtype,
+                                      index_value=out_series.index_value, nsplits=(tuple(nsplits),),
+                                      chunks=out_chunks)
+
+    @classmethod
+    def execute(cls, ctx, op):
+        df = ctx[op.inputs[0].key]
+        ctx[op.outputs[0].key] = df[op.labels]
+
+
+class DataFrameIndex(DataFrameOperand, DataFrameOperandMixin):
+    _op_type_ = OperandDef.INDEX
+
+    _col_names = AnyField('col_names')
+
+    def __init__(self, col_names=None, object_type=ObjectType.series, **kw):
+        super(DataFrameIndex, self).__init__(_col_names=col_names, _object_type=object_type, **kw)
+
+    @property
+    def col_names(self):
+        return self._col_names
+
+    def __call__(self, df):
+        # if col_names is a tuple or list, return a DataFrame, else return a Series
+        if isinstance(self._col_names, (list, tuple)):
```

Comment:
The `_col_names` won't be `tuple` (to be consistent with pandas).

Reply:
Fixed.